frostsnap_coordinator/src/bitcoin/wallet_reserved_spks.rs

```diff
@@ -0,0 +1,130 @@
+use std::collections::{BTreeSet, HashSet};
+
+use bdk_chain::{
+    bitcoin::{Script, ScriptBuf},
+    rusqlite_impl::migrate_schema,
+    Impl, Merge,
+};
+use rusqlite::named_params;
+
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Default)]
+pub struct ChangeSet {
+    spks: BTreeSet<(u64, ScriptBuf, bool)>,
```

Comment:
could comment that this bool is `is_add` to add/remove changes to changeset

Reply:
"reserve/unreserve spks".